src/js/components/search/topFilterBar/filterGroups/AgencyFilterGroup.jsx

```diff
@@ -43,6 +43,13 @@ export default class AgencyFilterGroup extends React.Component {
 
         agencies.forEach((value) => {
             let agencyTitle = value.subtier_agency.name;
+
+            if (value.agencyType === 'subtier') {
```

Comment:
Since the type is coming from `Agency` as one of two items in a static array, is it possible that it could be empty?